problem/models.py

```diff
@@ -114,11 +118,17 @@ class Submission(models.Model):
         (CPP, 'C++'),
         (CPP11, 'C++11'),
     )
```

Comment:
Could you remove such kind of these code at the beginning of class.
I think that will be more organized.